components/withSidebarCrossLinks.tsx

```diff
@@ -12,28 +12,39 @@ const WithSidebarCrossLinks: FC<WithCrossLinksProps> = ({ navKey }) => {
 
   const [[, sidebarNavigation]] = getSideNavigation([navKey]);
 
+  if (!sidebarNavigation || !sidebarNavigation.items) {
```

Comment:
Is this really needed? I doubt this would eve be null/undefined + if there are no items I assume nothing would really happen?

Reply:
since getSideNavigation depends on a [navKey] in a case where that gets wrong value it'd return an empty array, i'm not a 100% sure thats how can check for an empty array but i think its a good precaution since everything else after it depends on sidebarNavigation existing